src/cryptography/hazmat/backends/openssl/ocsp.py

```diff
@@ -319,6 +321,18 @@ def serial_number(self):
     def extensions(self):
         return _OCSP_BASICRESP_EXT_PARSER.parse(self._backend, self._basic)
 
+    @utils.cached_property
+    @_requires_successful_response
+    def single_extensions(self):
+        if self._backend._lib.CRYPTOGRAPHY_OPENSSL_110_OR_GREATER:
+            return _OCSP_SINGLERESP_EXT_PARSER.parse(
+                self._backend, self._single
+            )
+        else:
+            return _OCSP_SINGLERESP_EXT_PARSER_NO_SCT.parse(
```

Comment:
This branch isn't taken with current tests. We should add another test vector for an extension supported in < 1.1.0 (perhaps CRLReason) that will take this branch.